src/ai-bundle/src/Profiler/DataCollector.php

```diff
@@ -52,7 +57,6 @@ public function __construct(
 
     public function collect(Request $request, Response $response, ?\Throwable $exception = null): void
     {
-        $this->lateCollect();
```

Comment:
why this change?

Reply:
It seemed redundant to call late collect since it will be called on `kernel.terminate` so i've remove it, do you think it's relevant ?

Reply:
It was introduced in #173 with the reason - does this change break the profiler on streaming?